docs/run-platform-server/configuring.mdx

```diff
@@ -8,7 +8,14 @@ import { CodeExample } from "@site/src/components/CodeExample";
 
 Pre-requisites:
 
-- [Installation](./installing.mdx) is completed on a host machine and results in two executables: `stellar-horizon` and `stellar-core`. Per the installation instructions, `stellar-core` should be included in your host OS' `$PATH`, as the configurations below result in Horizon searching for a `stellar-core` to execute.
+- You have identified the [Installation](./installing.mdx) method for host system.
+
+  For bare metal, you have the two executables: `stellar-horizon` and `stellar-core` both on host OS' `$PATH`.
+
+  For running image, using the [stellar/stellar-horizon](https://hub.docker.com/r/stellar/stellar-horizon) direct on `docker run`, you have pulled the stellar/stellar-horizon image via `docker pull stellar/stellar-horizon#<tag>` onto host.
```

Comment:
```suggestion
  For running image, using the [stellar/stellar-horizon](https://hub.docker.com/r/stellar/stellar-horizon) direct on `docker run`, you have pulled the stellar/stellar-horizon image via `docker pull stellar/stellar-horizon#<tag>` onto host.
```
```suggestion
  For running image, using the [stellar/stellar-horizon](https://hub.docker.com/r/stellar/stellar-horizon) direct on `docker run`, you have pulled the stellar/stellar-horizon image via `docker pull stellar/stellar-horizon:<tag>` onto host.
```

Reply:
>  using the [stellar/stellar-horizon](https://hub.docker.com/r/stellar/stellar-horizon) direct on `docker run`

Not sure I understood this. Are we saying that the docker run will also pull the image?

> you have pulled the stellar/stellar-horizon image via `docker pull stellar/stellar-horizon#<tag>` onto host.

Would it be helpful to include the docker pull command in the installing doc?